File: 202101070257 - gamzee1256 (CSharp - Tez Automation)/03_example-code/GazeteOtomasyon/Form_Giris.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace GazeteOtomasyon
{
    public partial class Form_Giris : Form
    {
        public Form_Giris()
        {
            InitializeComponent();
        }
        Class_Islemler islemler = new Class_Islemler();
        private void btn_Giris_Click(object sender, EventArgs e)
        {
            if (txt_KullaniciAd.Text != "" && txt_Parola.Text != "")
            {
                if (islemler.Yonetici_Kontrol(txt_KullaniciAd.Text, txt_Parola.Text))
                {
                    MessageBox.Show("Hoşgeldiniz", "Bilgi", MessageBoxButtons.OK, MessageBoxIcon.Information);
                    Form form = new Form_Anasayfa();
                    form.Show();
                    this.Hide();
                }
                else
                    MessageBox.Show("Hatalı Kullanıcı Adı veya Parola", "Hata", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
            else
                MessageBox.Show("Gerekli alanladı doldurunuz !!!", "Uyarı", MessageBoxButtons.OK, MessageBoxIcon.Warning);
            
        }

        private void Form_Giris_Load(object sender, EventArgs e)
        {
            if (!islemler.BaglantiTest())
            {
                MessageBox.Show("Veritabanı bağlantı yapılandırması gerekli !!!", "Dikkat", MessageBoxButtons.OK, MessageBoxIcon.Warning);
                Form form = new Form_VeritabaniBaglanti();
                form.ShowDialog();
                Application.Restart();
            }
        }
    }
}

File: 202101240037 - Kielmoritz (VbNet - Cargo Automation)/01_source-code/05_project/VBOtomasyon/VBOtomasyon/Form_Giris.vb
Public Class Form_Giris
    Dim islemler As Class_Islemler = New Class_Islemler
    Private Sub llSifremiUnuttum_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles llSifremiUnuttum.LinkClicked
        If txtKullaniciAd.Text = "" Then
            MsgBox("Kullanıcı adını giriniz !")
            Return
        End If
        If islemler.Kullanici_Kontrol(txtKullaniciAd.Text) = False Then
            MsgBox("Kullanıcı Adı bulunamadı")
            Return
        End If
        Dim sifreYenile As Form_SifreYenile = New Form_SifreYenile
        sifreYenile.kullaniciAd = txtKullaniciAd.Text
        sifreYenile.ShowDialog()
    End Sub

    Private Sub btnGiris_Click(sender As Object, e As EventArgs) Handles btnGiris.Click
        If txtKullaniciAd.Text = "" Or txtParola.Text = "" Then
            MsgBox("Gerekli alanları doldurunuz")
            Return
        End If
        Dim md5 As String = islemler.Md5Uret(txtParola.Text)
        If islemler.Kullanici_Giris(txtKullaniciAd.Text, md5) Then
            MsgBox("Giriş başarılı")
            Dim anasayfa As Form_Anasayfa = New Form_Anasayfa
            anasayfa.Show()
            Me.Hide()
        Else
            MsgBox("Kullanıcı adı veya parola hatalı")
        End If
    End Sub
End Class